pallets/parachain-staking/src/weights.rs

```diff
@@ -140,27 +140,27 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 			.saturating_add(T::DbWeight::get().writes(4 as Weight))
 	}
 	fn nominate() -> Weight {
-		(109_000_000 as Weight)
+		(120_000_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(9 as Weight))
 			.saturating_add(T::DbWeight::get().writes(6 as Weight))
 	}
 	fn leave_nominators() -> Weight {
-		(113_000_000 as Weight)
-			.saturating_add(T::DbWeight::get().reads(8 as Weight))
-			.saturating_add(T::DbWeight::get().writes(6 as Weight))
+		(1_929_000_000 as Weight)
```

Comment:
So the first measurement came when we were leaving after having only made one nomination.

The new measurement came when we left after making the maximum number of nominations per account.

This provides support to limit each AccountId to a single delegation + pallet_proxy for multiple nomination functionality when required. This will be pursued in a follow-up PR after the v0.8 release.